src/components/SectionWidgets.tsx

```diff
@@ -79,15 +80,19 @@ function AboutOfficialRemotes() {
       <div className="flex justify-start">
         <div>
           <span>Kore™ (Android)</span>
-          <img
+          <StaticImage
             className="w-200 h-200"
             alt="Kore QR Code"
-            src="/images/kore-qr.webp"
```

Comment:
I don't understand why, but somehow this change causes the remote name to show up below the image and not lined up.
<img width="1505" alt="Screen Shot 2021-03-28 at 7 04 25 AM" src="https://user-images.githubusercontent.com/1402018/112760829-0a8ee200-8f94-11eb-9783-79225e5689d9.png">

Reply:
Fixed